cli/tests/tobs-cli-tests/installation_test.go

```diff
@@ -171,39 +178,51 @@ func testHelmShowValues(t testing.TB) {
 }
 
 func TestInstallation(t *testing.T) {
-	if testing.Short() {
+	v := os.Getenv("SKIP_INSTALL_TESTS")
+
+	if testing.Short() || v != "" {
```

Comment:
we should be conservative to make sure we don't skip tests
```suggestion
	if testing.Short() || v == "TRUE" {
```